src/Agriculture.jl

```diff
@@ -36,7 +36,7 @@ function run_timestep(s::Agriculture, tt::Int)
     d = s.Dimensions
 
     for rr in d.regions
-        v.allirrigation[rr, tt] = p.othercropsirrigation[rr, tt]
+	    v.allirrigation[rr, tt] = p.othercropsirrigation[rr, tt] + p.uniirrigation[rr, tt] + p.irrirrigation[rr, tt]
```

Comment:
Can you keep the indentation consistent?  You might be using an editor that uses tabs, rather than spaces.